### PR TITLE
Create project repo initial setup

### PR DESCRIPTION
**Pull request by jackbsteinberg**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  'env': {
+    'commonjs': true,
```
Comment (domenic) [inline .eslintrc.js]: Looks like this is redundant with `'node': true`

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/node_modules
```
Comment (domenic) [inline .gitignore]: Nit: newline at end of file

File: README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+# LAPI Rewriter Script
```
Comment (domenic) [inline README.md]: Get-originals Rewriter Codemod?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "lapi-rewriter",
+  "version": "1.0.0",
+  "description": "Rewriter presubmit script for LAPIs",
```
Comment (domenic) [inline package.json]: Name and description update to focus on get-originals.

File: package.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "lapi-rewriter",
+  "version": "1.0.0",
```
Comment (domenic) [inline package.json]: I prefer 1.0.0-pre until you actually do `npm publish`

File: package.json [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  "description": "Rewriter presubmit script for LAPIs",
+  "main": "lib/index.js",
+  "directories": {
```
Comment (domenic) [inline package.json]: `directories` doesn't actually do anything, so best to remove it

File: package.json [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    "lib": "lib"
+  },
+  "dependencies": {
+    "jest": "^24.8.0",
```
Comment (domenic) [inline package.json]: I think jest is a dev dependency?

File: package.json [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "devDependencies": {
+    "eslint": "^6.1.0",
+    "eslint-config-google": "^0.13.0"
+  },
+  "scripts": {
+    "start": "node lib/index.js",
+    "test": "jest"
+  },
+  "author": "jacksteinberg@chromium.org",
```
Comment (domenic) [inline package.json]:
```suggestion
  "author": "Jack Steinberg <jacksteinberg@chromium.org>",
```

File: .eslintrc.js (hunk 1 above)
Comment (jackbsteinberg) [reply .eslintrc.js]: This file was autogenerated (I picked Google's JS standard, which I assume we theoretically adhere to), I was inclined not to mess with it but I can remove this if you want.

File: package.json (hunk 7 above)
Comment (jackbsteinberg) [reply package.json]: This was also autogenerated, I didn't install it as --save-dev though, so maybe I should go back and do that.

File: package.json [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+    "eslint": "^6.1.0",
+    "eslint-config-google": "^0.13.0",
+    "jest": "^24.8.0"
+  },
+  "scripts": {
+    "start": "node lib/index.js",
+    "test": "jest"
+  },
+  "author": "Jack Steinberg <jacksteinberg@chromium.org>",
+  "repository": "https://github.com/jackbsteinberg/lapi-rewriter.git"
```
Comment (domenic) [inline package.json]:
```suggestion
  "repository": "https://github.com/jackbsteinberg/get-originals-rewriter.git"
```

File: package.json [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+  "dependencies": {
+    "jscodeshift": "^0.6.4"
+  },
+  "devDependencies": {
+    "eslint": "^6.1.0",
+    "eslint-config-google": "^0.13.0",
+    "jest": "^24.8.0"
+  },
+  "scripts": {
+    "start": "node lib/index.js",
```
Comment (domenic) [inline package.json]: Looks like this isn't useful either; best to remove

**Comment by jackbsteinberg** [comment]
> Nooo the test script is good, just the start one

Oh, I figured if we don't need `npm start` to run the rewriter itself why would we need `npm test` to run the tests when just writing `jest` will do the trick

**Comment by domenic** [comment]
`npm test` is the standard way of running the tests, and is generally applicable, but `npm start` is more meant for servers. Having `npm start` doesn't buy us much here because people won't use this by cloning the repo then doing `npm start`. Instead they'd use it more like https://github.com/jongold/import-codemod#usage (presumably without the config file).

**Comment by domenic** [comment]
When merging, don't forget to squash and merge. (You may want to use the repository settings to turn off the other possibilities, or at least the "Create a merge commit" option.)

File: .eslintrc.js [hunk 11]
```diff
@@ -0,0 +1,18 @@
+module.exports = {
```
Comment (fergald) [inline .eslintrc.js]: Is there a reason we're not using the same eslintrc as the other projects?

**Pull request merged**